atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -104,6 +101,7 @@ public static SerializableTransactionManager create(
             Set<Schema> schemas,
             Environment env,
             boolean allowHiddenTableAccess) {
+        DebugLogger.logger.warn("Called TransactionManagers.create. This should only happen once.");
```

Comment:
We should also mention what thread it's from, right? Just like `ServiceDiscoveringAtlasSupplier`